.gitignore

```diff
@@ -1,2 +1,2 @@
-target/
-.idea
+**/target/
+**/images/
```

Comment:
Why do you ignore "images"? Will the image still be available?

Reply:
I wasnt sure if I should include images or other media as they could perhaps lead to the repo being unnecessarily large, and also didnt really seem to belong to a code repo? Would you recommend to include the images as well?

Reply:
The reason I asked is if you need to deploy the website, the image links will be broken and can't be displayed. It is okay to commit images to code repo, especially for websites.

Alternatively, you can deploy the images elsewhere, e.g. CDN.

Reply:
@leonyeo, you can click the "Resolve conversation" if you have fixed this particular comment.